modules/tinymce/src/themes/silver/main/ts/ui/skin/Loader.ts

```diff
@@ -14,26 +15,54 @@ const loadStylesheet = (editor: Editor, stylesheetUrl: string, styleSheetLoader:
   return styleSheetLoader.load(stylesheetUrl);
 };
 
-const loadUiSkins = (editor: Editor, skinUrl: string): Promise<void> => {
-  const skinUiCss = skinUrl + '/skin.min.css';
-  return loadStylesheet(editor, skinUiCss, editor.ui.styleSheetLoader);
+const loadRawCss = (editor: Editor, key: string, css: string, styleSheetLoader: StyleSheetLoader): void => {
+  // Ensure the stylesheet is cleaned up when the editor is destroyed
+  editor.on('remove', () => styleSheetLoader.unloadRawCss(key));
+  return styleSheetLoader.loadRawCss(key, css);
+};
+
+const loadUiSkins = async (editor: Editor, skinUrl: string): Promise<void> => {
+  const skinUiCss = skinUrl + '/skin.css';
+  const css = await tinymce.Resource.get(skinUiCss);
+
+  if (Type.isString(css)) {
+    loadRawCss(editor, skinUiCss, css, editor.ui.styleSheetLoader);
+    return Promise.resolve();
```

Comment:
You can probably just `return` now that this is an async function 🤔 

`styleSheetLoader.loadRawCss()` doesn't return anything, so I'm surprised this worked before 🤔